new-log-viewer/src/App.tsx

```diff
@@ -1,7 +1,90 @@
+import {
+    CssVarsProvider,
+    extendTheme,
+} from "@mui/joy/styles";
+
+import KeyboardArrowDown from "@mui/icons-material/KeyboardArrowDown";
+
 import Layout from "./components/Layout";
 import StateContextProvider from "./contexts/StateContextProvider";
 import UrlContextProvider from "./contexts/UrlContextProvider";
+import {CONFIG_KEY} from "./typings/config";
+import {CONFIG_DEFAULT} from "./utils/config";
+
 
+const monacoTheme = extendTheme({
```

Comment:
Can we move this into another source file?

Reply:
Moved to `utils/theme.tsx`.

Reply:
Are the changes pushed?